examples/EmptyLogger/EmptyLogger.ino

```diff
@@ -0,0 +1,67 @@
+#include <Arduino.h>
+#include <string.h>
+
+#include <RF24Logger.h>
+#include <RF24StreamLogAppender.h>
+
+// Define global vendor id (it is stored in flash memory)
+const char vendorID[] PROGMEM = "RF24LogExample";
+
+// DO NOT create hardware serial port log appender
+// RF24StreamLogAppender rf24SerialLogAppender(&Serial);
+
+// Need to remember that backslash must be escaped in the string text
+// Banner generated with https://devops.datenkollektiv.de/banner.txt/index.html
+const char banner0[] PROGMEM = " ___   ___   ___   _ _    _                 ";
+const char banner1[] PROGMEM = "| _ \\ | __| |_  ) | | |  | |     ___   __ _ ";
+const char banner2[] PROGMEM = "|   / | _|   / /  |_  _| | |__  / _ \\ / _` |";
+const char banner3[] PROGMEM = "|_|_\\ |_|   /___|   |_|  |____| \\___/ \\__, |";
+const char banner4[] PROGMEM = "                                      |___/ ";
+
+void setup()
+{
+   // configure serial port baudrate
+   Serial.begin(57600);
+   // DO NOT set serial port appender
+   // rf24Logger.setAppender(&rf24SerialLogAppender);
+
+   Serial.println("There is no appender defined. Nothing should be logged.");
+
+   // display banner
+   rf24Logger.info((const __FlashStringHelper*) vendorID,

```

Comment:
Maybe the library could contain an overloaded function that accepts this type?

Reply:
Oops, I do not understand this sentence.

Reply:
Instead of forcing the user to type cast the `vendorId`, you could overload the function to accept the datatypes `string` or `const char *`. This is why I used templated datatypes on my proposal.

Reply:
AH, now I see. Thanks.